internal/output/json_test.go

```diff
@@ -170,21 +171,21 @@ func TestJSON_Error(t *testing.T) {
 		expectedErr string
 	}{
 		{
-			"error string",
-			args{errors.New("hello")},
-			`{"errors":["hello"],"code":1}` + "\x00\n",
+			"localized error",
+			args{locale.NewError("", "hello")},
```

Comment:
`errs.New()` would be invalid as we only want to expose localized errors. As it stands if you pass a non-localized error it will panic if run from a local build, and log to rollbar if run from a CI build, whilst reporting the topmost error. That's probably not worth testing at this level though.